### PR TITLE
feat: list

### PR DESCRIPTION
**Pull request by SuyashTiwariCoder**
added:
1) spaced(normal lists)
2) stacked (notification Page)

### CODE REVIEW

**Review by sks-21 — APPROVED** [review]
The code looks nice. Good to merge.

File: components/List/list.html [hunk 1]
```diff
@@ -0,0 +1,166 @@
...
+<div class="sidebar">
+
+    <h3>Get Started</h3>
+    <div class="divider"></div>
+    <ul>
+    <li><a href="/Documentation/documentation.html">Introduction</a></li>
+    <li><a href="/Utilities/Colors/color.html">Colors</a></li>
+    <li><a href="/Utilities/Text Utility/textUtil.html">Text Utility </a></li>
+    <li><a href="/Utilities/Conclusion/conclusion.html">Conclusion</a></li>
+
```
Comment (Priya-Patel31) [inline components/List/list.html:35]: remove unnecessary space

**Review by Priya-Patel31 — APPROVED** [review]
Your code code and UI both looks clean and seems good . So you can go ahead and merge it !!

File: components/List/list.html (hunk 1 above)
Comment (SuyashTiwariCoder) [reply components/List/list.html:35]: ok I will refrator it .

**Pull request merged**